### ISSUE TITLE
Add option to remove the title bar

### ISSUE THREAD

**Issue by GoogleCodeExporter** [mintty/mintty#7]
```
This would take the minimalist approach to the extreme by leaving only the
terminal screen and the window border. Alternative ways to move or close
the window would be needed though.
```

Original issue reported on code.google.com by `andy.koppe` on 30 Dec 2008 at 11:54

**Comment by mintty** [comment]
Added command line options -B frame and -B void.

**Comment by cebaa** [comment]
Is it possible to specify this in `~/.minttyrc` somehow? I tried `Border=void`, but that did not seem to work.

**Comment by mintty** [comment]
No, it's a command-line only option, `mintty --Border=void` (not `mintty -o Border=void`).

**Comment by cebaa** [comment]
Is there a technical reason for it to not be available in `.minttyrc`? Asking because this requires this to be put in a batch file that cannot be pinned to Windows taskbar (and workarounds seem to make the pin separate from the program instance icon, so not user friendly that way).

**Comment by mintty** [comment]
There's not a technical reason, rather a use case consideration. The assumption was that normal invocation should have borders, to provide seamless access to windows functions, while borderless mode would only be desired for special cases.

**Comment by cebaa** [comment]
If possible, adding this as a config option in some of the subsequent releases would really help people like many on this thread that are using that as a default. It avoids having to deal with other issues as a result.

**Comment by mintty** [comment]
So, just to clarify: you want stripped borders for all your mintty sessions?

**Comment by cebaa** [comment]
Yeah, that's how I'm running them now: 3 monitors, one window on each, no borders on any of the windows, multiple tabs in each window. The reason is mainly that I have no use of the title bar, so stripping the borders (which removes the title bar) allows for more usable space.

**Comment by mintty** [comment]
Released 3.7.5.

**Comment by derekschrock** [comment]
Should "Border" be "BorderStyle" now?

https://github.com/mintty/mintty/commit/8a0f405d9d91fc84e01ed2a0ac94c7695d76988b#diff-3ac07983d5ee15d76c83bf2cec2b7c147383c985413b8673871951884b098893R6569

It seems all usage of -B from the command line is now an unknown option:

```
$ mintty.exe -B normal
Ignoring unknown option 'Border'.

$ mintty.exe -B thin
Ignoring unknown option 'Border'.

$ mintty.exe -B void
Ignoring unknown option 'Border'.
```

**Comment by mintty** [comment]
Outch! Thank you. Fixed, and there'll be another release soon anyway.

**Comment by mintty** [comment]
Released 3.7.6.
